Task3/Task3_Nikolay_Varadinov/src/python/car_wash/lib_py_pool/py_pool.py

```diff
@@ -0,0 +1,151 @@
+"""Module with Read and Write Connection Pool."""
+
+import logging
+import time
+import typing
+
+from contextlib import contextmanager
+from sys import exc_info
+from threading import Lock
+
+import psycopg2
+
+from . meta_classes import SingletonMeta
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class PoolConn(metaclass=SingletonMeta):
+
+    """Pool which is python list and strores DB connections."""
+
+    @staticmethod
+    def stringer(db_config: dict) -> str:
```

Comment:
Not so good idea. You have dictionary as constant. Please use it . We do not need to convert dict into string.